src/controllers/scoresController.ts

```diff
@@ -25,10 +25,10 @@ async function getScoresFromDB(gameIndex: number) {
       `SELECT * 
       FROM kubercade.high_score_table
       WHERE game_index=$1
-      ORDER BY score DESC`,
+      ORDER BY datetime DESC, score DESC;`,
```

Comment:
shouldn't this be ordered by score first?

Reply:
My bad, I thought it was like sheets sorting where the last sort has the highest priority, but looking at sql docs again you are right. Fixed in https://github.com/knative-portability/Kubercade/pull/15/commits/1a7eb59810b5b408991a30849ac4e5a7db3cab00